horace_core/sqw/PixelData/@PixelTmpFileHandler/PixelTmpFileHandler.m

```diff
@@ -0,0 +1,112 @@
+classdef PixelTmpFileHandler
+
+properties (Constant, Access=private)
+    DIRTY_PIX_DIR_BASE_NAME_ = 'sqw_pix%05d';
```

Comment:
do these need to be `DIRTY_`? This temp file IO class just deals with temporary files.